Portfolio/src/App.vue

```diff
@@ -3,96 +3,95 @@ import { ref } from 'vue';
-                        <RouterLink style="text-decoration: none; color: #1e1e1e; ;" to="/">Home
-                        </RouterLink>
-                    </li>
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e; " to="/work">Work</RouterLink>
-                    </li>
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e;" to="/about">About</RouterLink>
-                    </li>
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e;" to="/contact">Contact</RouterLink>
-                    </li>
-                </ul>
-            </nav>
-
-        </div>
-    </header>
-
-    <div v-if="isMenuShown" class="openedMenu" style="font-family: 'RalewayBold'; text-transform: uppercase;">
-        <ul class="menu-links">
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff; ;" to="/">Home
-                </RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff; " to="/work">Work</RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff;" to="/about">About</RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff;" to="/contact">Contact</RouterLink>
-            </li>
-        </ul>
-
-        <div class="menu-fullName">
-            <h1 class="menu-firstName">{{ homeViewContent.firstName }}</h1>
-            <h1 class="menu-lastName">{{ homeViewContent.lastName }}</h1>
+    <Wrapper>
+        <header>
+            <div class="router-wrapper">
+                <button @click="isMenuShown = !isMenuShown" class="menuButton">
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }">
+                    </div>
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }"></div>
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }"></div>
+                </button>
+
+                <nav>
+                    <ul class="navbar">
+                        <li>
+                            <RouterLink to="/">Home
+                            </RouterLink>
+                        </li>
+                        <li>
+                            <RouterLink to="/work">Work</RouterLink>
+                        </li>
+                        <li>
+                            <RouterLink to=" /about">About</RouterLink>
+                        </li>
+                        <li>
+                            <RouterLink to="/contact">Contact
+                            </RouterLink>
+                        </li>
+                    </ul>
+                </nav>
+
+            </div>
+        </header>
+
+        <div v-if="isMenuShown" class="openedMenu" style=" text-transform: uppercase;">
```

Comment:
I would highly recomment keeping the styles in their file and never use inline. Because inline override the other kinds of styles, so it would mess up with the rest of the code. U really need to read this:

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance

Reply:
Also something I noticed is that above u have a nav wrapper in a header and inside them u have ur navigation, but down here u have a list of links that is wrapped in neither a header nor a nav.